src/components/EarnReport.tsx

```diff
@@ -37,26 +49,25 @@ const YieldgenReportTable = ({ lines, maxAmountOfRows = 15 }: YielgenReportTable
             <thead>
               <tr>
                 {headers.map((name, index) => (
-                  <th key={`header_${index}`}>{name}</th>
+                  <th key={`header_${index}`}>{reportHeadingMap[name] || name}</th>
                 ))}
               </tr>
             </thead>
             <tbody>
               {visibleLines.map((line, trIndex) => (
                 <tr key={`tr_${trIndex}`}>
                   {line.map((val, tdIndex) => (
-                    <td key={`td_${tdIndex}`}>{val}</td>
+                    <td key={`td_${tdIndex}`}>
+                      {headers[tdIndex] && headers[tdIndex].includes('satoshi') ? (
```

Comment:
nit: It feels like a dedicated dictionary containing the formatting would be more robust to future changes/additions.. e.g.
```
reportHeadingMap = {
    'cj amount/satoshi': {
      heading: t('earn.report.heading_cj_amount'),
      format: 'balance'
    }, 
    'my input count': {
      heading: t('earn.report.heading_input_count'),
      format: 'number'
    },
    'notes\n': {
      heading: t('earn.report.heading_notes'),
      format: 'string'
    },
    [...]
}
```

But, as it is not likely to change soon or often, I guess it is okay.

Reply:
Good point! Added.